contrib/kube-redis-exporter-with-redis-sentinel.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: sentinel
+    role: service
+  name: redis-sentinel
+spec:
+  ports:
+    - port: 26379
+      targetPort: 26379
+  selector:
+    redis-sentinel: "true"
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: redis
+spec:
+  serviceName: redis-sentinel
+  replicas: 1
+  selector:
+    matchLabels:
+      name: sentinel
+  template:
+    metadata:
+      labels:
+        name: sentinel
+        redis-sentinel: "true"
+#      annotations:
+#        security.alpha.kubernetes.io/unsafe-sysctls: net.core.somaxconn=1024
+    spec:
+      containers:
+      - name: redis
+        image: cookingcodewithme/redis:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 6379
+        resources:
+          limits:
+            cpu: "0.5"
+#        volumeMounts:
+#        - mountPath: /redis-master-data
+#          name: data
+      - name: sentinel
+        image: cookingcodewithme/redis:latest
+        imagePullPolicy: Always
+        env:
+          - name: SENTINEL
+            value: "true"
+        ports:
+          - containerPort: 26379
+#      volumes:
+#        - name: data
+#          emptyDir: {}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: default
+  name: redis-exporter
+  labels:
+    name: redis-exporter
+spec:
+  ports:
+  - name: redis-exporter
+    port: 9121
+  selector:
+    name: redis-exporter
+
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  namespace: default
+  name: redis-exporter
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: redis-exporter
+    spec:
+      containers:
+        - name: redis-exporter
+          image: cookingcodewithme/redis_export:latest
+          imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: "0.1"
```

Comment:
This could probably be even less though tbh, I haven't looked at actual CPU requirements in a while.